packages/base-map/src/index.story.tsx

```diff
@@ -129,7 +120,9 @@ export const withSampleMarkers = () => (
 
 export const overlayWithLargeDataSet = () => (
   <>
-    <div>Do not add Storybook overhead on layers with large dataset...</div>
+    <div style={{ position: "relative", zIndex: 1000 }}>
```

Comment:
The "Click-to-set-bounds" story needs something like this too, and probably you need to say for that story and this one that they need no decorator (otherwise, two maps get rendered!)